### ISSUE TITLE
Issues with SSL

### ISSUE THREAD

**Issue by pdubb-sauce** [humanmade/Custom-Meta-Boxes#454]
Hi,

I understand this library is no longer in development, however I'm hoping someone can help me. 

I'm having issues with CMB loading properly in a secured SSL environment. For some reason 3 CMB includes are not loading over https and instead stuck in http:

The 3 stylesheets id's are: cmb-styles-css, select2-css, cmb-jquery-ui-css

If anyone can point me in the right direction to get this corrected that would be awesome, thanks in advance.

**Comment by mikeselander** [comment]
@pdubb-sauce I'm a little lost myself on what could be causing this issue. We're using core WordPress functions to build the URL for loading the scripts and styles. 

Are the JS files also causing mixed-content issues? Could you double-check that you've got the URL properly set with `https` in all applicable options rows?

**Comment by pdubb-sauce** [comment]
@mikeselander Yeah I'm baffled. The JS files for cmb and select2 are both loading over https.

Can you please elaborate a little more on the setting of https to all applicable options rows?

**Comment by mcjwb** [comment]
I know this is a bit old, but I just had this issue and I traced it back to a core option called "siteurl" stored in the database (wp_options table). Mine was not set to "https". changing this fixed the issue for me.
